packages/test/service-load-test/src/nodeStressTest.ts

```diff
@@ -32,13 +33,14 @@ const codeDetails: IFluidCodeDetails = {
 
 const codeLoader = new LocalCodeLoader([[codeDetails, fluidExport]]);
 
-function createLoader(testDriver: ITestDriver) {
+function createLoader(testDriver: ITestDriver, runId?: number) {
     // Construct the loader
     const loader = new Loader({
         urlResolver: testDriver.createUrlResolver(),
         documentServiceFactory: testDriver.createDocumentServiceFactory(),
         codeLoader,
-        logger,
+        logger: ChildLogger.create(logger, undefined, {all: {buildId: process.env.BUILD_BUILDID,
```

Comment:
we can check this in, but i really think this should just be confgured in the logger we inject by default

Reply:
Good idea, added a task over here:  https://dev.azure.com/fluidframework/internal/_backlogs/backlog/internal%20Team/Issues/?workitem=9

There's a few clean-up items in that backlog, I'll open a GitHub issue for April to cover those.

Reply:
https://github.com/microsoft/FluidFramework/issues/5567